py/test/selenium/webdriver/common/click_tests.py

```diff
@@ -35,3 +35,6 @@ def testCanClickOnALinkThatOverflowsAndFollowIt(driver):
 def testClickingALinkMadeUpOfNumbersIsHandledCorrectly(driver):
     driver.find_element(By.LINK_TEXT, "333333").click()
     WebDriverWait(driver, 3).until(EC.title_is("XHTML Test Page"))
+
+def testCannotClickDisabledButton(driver):
+    WebDriverWait(driver, 3).until(EC.element_to_be_unclickable(By.ID, "disabled-button"))
```

Comment:
I believe it's misleading name for the condition. I prefer "element_to_be_disable"
We can have a condition, when element is enabled but we can't click it, because another element overlays above it.
So, If we use "unclickable" we might mislead people, who use that condition to verify if element can be clicked